crates/uv-distribution/src/lib.rs

```diff
@@ -35,3 +49,133 @@ impl From<Metadata23> for ArchiveMetadata {
         }
     }
 }
+
+#[derive(Debug, Error)]
+pub enum MetadataLoweringError {
+    #[error(transparent)]
+    Workspace(#[from] WorkspaceError),
+    #[error(transparent)]
+    Lowering(#[from] LoweringError),
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+#[serde(rename_all = "kebab-case")]
+pub struct Metadata23Lowered {
```

Comment:
Could we just call this `Metadata` or something, differentiated from `Metadata23` from the spec? Or use `pypi_types::Metadata23` everywhere?

Reply:
Absolutely these names are only for me to be easier to work with and suppose to change before merging, ideally we'd even remove some type